src/epcpm/parameterstointerface.py

```diff
@@ -303,21 +305,25 @@ def interface_variable_name(self):
         model = maybe_model
         model_variable = f"sunspecInterface.model{model.id}"
 
-        return f"&{model_variable}.{parameter.abbreviation}"
-
+        if reference == True:
+            return f"&{model_variable}.{parameter.abbreviation}"
+        else:
+            return f"{model_variable}.{parameter.abbreviation}"
```

Comment:
Consider replacing with arguably not easier to read one liner:
`return f"{'&' if reference else ''}{model_variable}.{parameter.abbreviation}"`